DataFormats/PatCandidates/interface/TriggerObjectStandAlone.h

```diff
@@ -33,8 +34,10 @@ namespace pat {
       /// Keeping the old names of the data members for backward compatibility,
       /// although they refer only to HLT objects.
 
-      /// Vector of labels of all HLT filters or names od L1 conditions the trigger objects has been used in
-      std::vector< std::string > filterLabels_;
+      /// Vector of labels of all HLT filters or names of L1 conditions the trigger objects has been used in
+      mutable std::vector< std::string > filterLabels_;
```

Comment:
well it is cleared before when packing, anyhow this topic of what we should write and when we should compress (what we should make mutable etc. etc..) is still to be finalized.